spec/classes/init_spec.rb

```diff
@@ -82,7 +82,8 @@
      processorcount: 3,
      use_sasl: true,
      large_mem_pages: true,
-     extended_opts: %w[lru_crawler lru_maintainer]
+     conn_backlog: 500,
```

Comment:
```
       -b <num>
              Set the backlog queue limit to <num> connections. The default is 1024.
```

Reply:
This is just a test case, not defaults. Defaults are in init.pp. Its defaulting to undef to retain backwards-compatibility w/ the interface.